test/Driver/linker-order.ll

```diff
@@ -0,0 +1,9 @@
+; RUN: llc -mtriple armv7--linux-gnueabihf -filetype obj -o - %s | swift-autolink-extract -o - - | FileCheck %s
```

Comment:
Can we get a comment  in the test case that actually describes the purpose of this test?  Right now it is unclear by just looking at the test to discern what it is testing.